apps/prairielearn/src/tests/sync/questionsSync.test.ts

```diff
@@ -262,6 +271,249 @@ describe('Question syncing', () => {
     assert.ok(syncedQuestionTag);
   });
 
+  it('syncs authors', async () => {
+    const courseData = util.getCourseData();
+    const newAuthor = {
+      name: 'Example',
+      email: 'example@example.org',
+      orcid: '0000-0000-0000-0001',
+    };
+
+    if (!courseData.questions[util.QUESTION_ID].authors) {
+      courseData.questions[util.QUESTION_ID].authors = [];
+    }
+    courseData.questions[util.QUESTION_ID].authors.push(newAuthor);
+    const courseDir = await util.writeCourseToTempDirectory(courseData);
+    await util.syncCourseData(courseDir);
+
+    const originalSyncedQuestion = await findSyncedQuestion(util.QUESTION_ID);
+    assert.ok(originalSyncedQuestion);
+
+    // Check that the author was added to the authors table
+    const authors = await util.dumpTableWithSchema('authors', AuthorSchema);
+    const author = authors.find(
```

Comment:
This `.find` appears to be used a couple times in the tests. Can you pull that out to a helper?

In general, I would try to modularize the tests to a so that each test essentially has "set of inputs -> helpers -> validate set of outputs". There isn't anything super actionable, this is probably good enough as is.